util/stmtsummary/statement_summary.go

```diff
@@ -388,20 +410,63 @@ func (ssMap *stmtSummaryByDigestMap) SetRefreshInterval(value string, inSession
 	}
 }
 
-func (ssMap *stmtSummaryByDigestMap) RefreshInterval() int64 {
+// refreshInterval gets the refresh interval for summaries.
+func (ssMap *stmtSummaryByDigestMap) refreshInterval() int64 {
 	return atomic.LoadInt64(&ssMap.sysVars.refreshInterval)
 }
 
+// SetHistorySize sets the history size for all summaries.
+func (ssMap *stmtSummaryByDigestMap) SetHistorySize(value string, inSession bool) {
+	ssMap.sysVars.Lock()
+	if inSession {
+		ssMap.sysVars.sessionHistorySize = value
```

Comment:
`ssMap.sysVars.sessionHistorySize` only used in this function, can we remove this?